docs/platform-integration/appmodel/maps.md

```diff
@@ -13,9 +13,40 @@ This article describes how you can use the .NET Multi-platform App UI (.NET MAUI
 
 The `IMap` and `Map` types are available in the `Microsoft.Maui.ApplicationModel` namespace.
 
+## Get started
+
+To access the browser functionality, the following platform-specific setup is required.
+
+<!-- markdownlint-disable MD025 -->
+# [Android](#tab/android)
+
+Android uses the `geo:` Uri scheme to launch the maps application on the device. This may prompt the user to select from an existing app that supports this Uri scheme. Google Maps supports this scheme.
+
+In the _Platforms/Android/AndroidManifest.xml_ file, add the following `queries/intent` nodes the `manifest` node:
```

Comment:
"add the following `queries/intent` nodes to the `manifest` node"

Reply:
```suggestion
In the _Platforms/Android/AndroidManifest.xml_ file, add the following `queries/intent` nodes to the `manifest` node:
```